Monika After Story/game/script-topics.rpy

```diff
@@ -15724,3 +15724,58 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="mas_not_working",
+            category=['you'],
+            prompt="I'm no longer working.",
+            pool=True,
+            conditional=("persistent._mas_has_work is True"),
+            aff_range=(mas_aff.NORMAL, None)
+
```

Comment:
I assume this requires an unlock action. As such, this shouldn't be managed by random pool unlocks, but by its conditional instead. Therefore, it needs a `no_unlock` rule.
```suggestion
            rules={"no_unlock": None}
```